modules/data-mover-for-csi-snapshots.adoc

```diff
@@ -0,0 +1,197 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+:_content-type: PROCEDURE
+[id="data-mover-for-csi-snapshots_{context}"]
+= Data Mover for CSI snapshots
+
+
+The OADP 1.1 Data Mover enables customers to back up CSI volume snapshots to a remote object store and to restore stateful applications from the store in the event of a failure, accidental deletion or corruption of the cluster.
+The OADP 1.1 Data Mover solution uses the Restic option of VolSync.
+
+[NOTE]
+=====
+Data Mover supports backup and restore of CSI volume snapshots only.
+=====
+
+.Prerequisites
+
+* Have a stateful application running in a separate namespace.
+
+* The `StorageClass` and `VolumeShapshotClass` CRs support CSI.
+
+* Only one `volumeSnapshotClass` CR has the annotation `snapshot.storage.kubernetes.io/is-default-class: true`.
+
+* Only one `storageClass` CR has the annotation `storageclass.kubernetes.io/is-default-class: true`.
+
+* Include the label `{velero-domain}/csi-volumesnapshot-class: 'true'` in your `VolumeSnapshotClass` CR to indicate for Velero which to use.
+
+* Install the VolSync Operator using the Operator Lifecycle Manager (OLM). Administrators can link:https://docs.openshift.com/container-platform/4.10/operators/admin/olm-adding-operators-to-cluster.html[install Operators on clusters]. Non-administrators can link:https://docs.openshift.com/container-platform/4.10/operators/user/olm-installing-operators-in-namespace.html[install Operators in namespaces].
++
+[NOTE]
+====
+The VolSync Operator is required only for use with the Technology Preview Data Mover. The Operator is not required for using OADP GA features.
+====
+
+* Install the link:https://github.com/openshift/oadp-operator/blob/master/docs/install_olm.md[OADP Operator] using OLM.
```

Comment:
please refer downstream links, like:
https://docs.openshift.com/container-platform/4.11/backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.html

Reply:
@mperetzred  We cannot use internal links or xrefs in modules, only in assemblies (for technical reasons). Actually, links to Github are discouraged too, if not completely forbidden. 
Here, I can just leave it without a link: 
Install the OADP Operator using OLM.

Reply:
+1 on using the downstream docs

Reply:
@sbeskin-redhat we want to refer the customer in the official docs to install it using official downstream images, and not by installing it with a makefile from upstream repo - which is a completely different workflow. It's always important to refer on the official docs to the correct installation path, which would be mostly documented on the official/downstream documentation.